sql/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java

```diff
@@ -239,6 +239,23 @@ public void testShowCreateTableWithGeneratedColumn() {
                   ")");
     }
 
+    @Test
+    public void testShowCreateTableWithDefaultClauseColumns() {
```

Comment:
or shall we merge this test with generated columns and name it `testShowCreateTableWithColumnExpressions`?

Reply:
I personally prefer a separate test.

Reply:
We could probably also use the `SqlExecutor` infrastructure to create a test that is a bit cheaper than a full integration test.

Reply:
replaced the `ShowIntegrationTest` test with the test for the `MetaDataToASTNodeResolver` that uses the SqlExecutor infrastructure.